spring-cloud-starter-stream-processor-aggregator/src/main/java/org/springframework/cloud/stream/app/aggregator/processor/ExcludeStoresAutoConfigurationEnvironmentPostProcessor.java

```diff
@@ -53,6 +56,7 @@ public void postProcessEnvironment(ConfigurableEnvironment environment, SpringAp
 						MongoRepositoriesAutoConfiguration.class.getName() + ", " +
 						EmbeddedMongoAutoConfiguration.class.getName() + ", " +
 						RedisAutoConfiguration.class.getName() + ", " +
+//					TODO	ClientCacheAutoConfiguration.class.getName() + ", " +
```

Comment:
I guess the local `ClientCacheAutoConfiguration` (e.g. the temporal copy from Jonh) is not a real a real auto-configuration (e.g not in the spring.factories) and this is the reason to exclude it here?

Reply:
Also, you can add 2018 to the license doc

Reply:
Well, technically yes, it is not a fully-blown auto-configuration since it isn't included into the `spring.factories`. Agreed. And that is the reason why it's not included here in excludes (cofused? :smile: ), but at the same time we will need it when `gemfire` profile is selected.

I would appreciate if you fix that `2018` during merge :wink: